app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Contributing Guide</h1>
```

Comment:
This works like a summary for multiple contributing guides, naming this will be tricky.